src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -80,7 +80,10 @@ export class QuerySummary extends Component {
      */
     noResultsFoundMessage: ComponentOptions.buildStringOption({
       defaultValue: l('noResultFor', '${query}'),
-      depend: 'enableNoResultsFoundMessage'
+      depend: 'enableNoResultsFoundMessage',
+      postProcessing: (value: string, options: IQuerySummaryOptions) => {
```

Comment:
Since we are not using the `options` arg, would it be possible to remove? or is there a typescript restriction? e.g.

```
postProcessing: (value: string) => escape(value)
```